requirements.txt

```diff
@@ -3,5 +3,6 @@ numpy==1.9.2
 scipy==0.16.0
 matplotlib==1.4.3
 nibabel==2.0.1
-scikit_learn==0.15.2
+scikit-learn==0.15.2
```

Comment:
We need scikit_learn with the underscore also we don't need textblob below. Please make sure ur file matches the old version since no changes are needed.